docker-compose.yml

```diff
@@ -12,6 +15,61 @@ services:
     depends_on:
       - postgrescms
 
+  redis:
```

Comment:
is there a reason to restructure the docker-compose?  I just want to make sure there isn't any change between redis/rabbitmq, postgrescms, Memcached and elastic search to review and its hard to see the diff when big structure changes occur.

Reply:
Yes, I had to ensure I had captured the stuff that core expects and this ordering is how the core compose files are structured. There are _zero_ changes to the code other than it got moved around in all the blocks that existed before. But I could not do a sane comparison between the two without a reordering. I don't know why we were not already aligned with the core organizational pattern to begin with, but now we are. Drives me crazy when codebases that are clones can not be compared because they aren't ordered correctly.

Reply:
so this change matches the order of Core-Portal's `server/conf/docker/docker-compose-dev.all.debug.yml` 👍 

if I could go back in time, I would even divide each service into it's own file to match Camino as we are mimicking that as well here.

Reply:
Yeah, seems like that was the intention but we never went all the way for some reason. We should follow up on that one day. =)